lib/web_console/middleware.rb

```diff
@@ -18,8 +18,9 @@ class Middleware
 
     UNACCEPTABLE_REQUEST_MESSAGE = "A supported version is expected in the Accept header."
 
-    cattr_accessor :whiny_requests
-    @@whiny_requests = true
+    cattr_accessor :verbose_logging, :acceptable_content_types
+    @@verbose_logging = true
+    @@acceptable_content_types = [Mime::HTML]
```

Comment:
This option should be in the Response class, right? Actually, do we need it to be configurable anymore? I never documented it for a reason – every case where a console wasn't shown, I treated like a bug, but gave the users an option to fix their problems in the mean time.